providers/types.go

```diff
@@ -33,3 +34,13 @@ type AuthType string
 const (
 	AuthTypeOAuth2 AuthType = "oauth2"
 )
+
+func (i *ProviderInfo) GetOption(key string) (string, bool) {
+	if i.ProviderOptions == nil {
```

Comment:
This if statement isn't needed, ` i.ProviderOptions[key]` will already return `"", false` if key is not found. See https://go.dev/play/p/fGhFnVJoZzy

Reply:
This is to check if the map even exists (it won't for hubspot which doesn't have any options)